tests/plugins/inlinetoolbar/positioning.js

```diff
@@ -56,7 +56,7 @@
 				assert.ignore();
```

Comment:
Doesn't it actually make sense to work directly on `inlineToolbarView` rather than create a full `inlineToolbar`?